libpod/networking_linux.go

```diff
@@ -63,15 +62,14 @@ func (r *Runtime) configureNetNS(ctr *Container, ctrNS ns.NetNS) (err error) {
 		}
 	}()
 
-	ctr.state.NetNS = ctrNS
-	ctr.state.NetworkStatus = make([]*cnitypes.Result, 0)
+	networkStatus := make([]*cnitypes.Result, 0)
```

Comment:
Nit: probably `,1)` is a bit faster, we expect to get at least 1 result

Reply:
done